src/lib.rs

```diff
@@ -1114,7 +1121,7 @@ decl_module! {
 
         /// Create new thread in category with poll
         fn create_thread(origin, forum_user_id: T::ForumUserId, category_id: T::CategoryId, title: Vec<u8>, text: Vec<u8>, labels: BTreeSet<T::LabelId>,
-            poll: Option<Poll<T::Moment>>,
+            poll: Option<Poll<T::Moment>>, sticky_index: Option<u32>,
```

Comment:
The simplest thing would be to just introduce `set_stickied_threads(origin,categor_id, stickied_ids: Vec<T::ThreadId>)`, and it could be used both for setting and reordering stickied threads by moderators.

Reply:
Accepted. It is my misunderstaning for the requirement. The method set_stickied_threads is clearer way for order sticky threads. I will implement it.